src/pages/components/input.mdx

```diff
@@ -39,6 +39,18 @@ import InputCode from '!!raw-loader!../../examples/forms/inputs/Input.tsx';
   <Input />
 </CodeExample>
 
+### Auto-expanding textarea
```

Comment:
The current example placement makes it seem like we provide an auto-expanding input. I would move this lower in position relative to the `Textarea input` section.

Reply:
I battled with this and I'm glad you pointed it out. Does seem to make more sense (and more discoverable) within other textarea documentation.